packages/components/src/components/Menu/Menu.test.tsx

```diff
@@ -36,4 +36,21 @@ describe("<Menu />", () => {
     expect(screen.getByText("Item 1")).toBeInTheDocument();
     expect(screen.getByText("Item 2")).toBeInTheDocument();
   });
+
+  it("allows a disabled prop for a menu item and disables the button", () => {
+    render(
+      <Menu items={[{ onClick: jest.fn(), label: "Item 1", disabled: true }]} />
```

Comment:
Small ignorable suggestion here. You could:

```
const onClick = jest.fn()

<Menu items={[{ onClick, label: "Item 1", disabled: true }]} />
```

then you can be sure that `onClick` is not called

Reply:
Agree with you, but I would create another test just to test the onClick prop and make sure it's only called when clicked.